src/Reference.php

```diff
@@ -195,8 +195,7 @@ protected function initTableAlias(): void
             $ourModel = $this->getOurModel();
 
             $this->table_alias = $this->link;
-            $this->table_alias = preg_replace('/_' . ($ourModel->id_field ?: 'id') . '/', '', $this->table_alias);
-            $this->table_alias = preg_replace('/([a-zA-Z])[a-zA-Z]*[^a-zA-Z]*/', '\1', $this->table_alias);
```

Comment:
@DarkSide666 @georgehristov what is the purpose of this regex replace? - one some MSSQL test is failing

plus for some reasons, stack trace is not shown in failing test with PHP 8:
```
1) Atk4\Data\Tests\SubTypesTest::testBasic
Undefined array key ":Ob"

/__w/data/data/vendor/atk4/dsql/src/Mssql/ExpressionTrait.php:48
/__w/data/data/vendor/atk4/dsql/src/Mssql/ExpressionTrait.php:52
/__w/data/data/vendor/atk4/dsql/src/Expression.php:656
/__w/data/data/src/Persistence/Sql.php:683
/__w/data/data/src/Persistence/Sql.php:714
/__w/data/data/src/Model.php:1131
/__w/data/data/src/Model.php:1156
/__w/data/data/src/Model.php:1578
/__w/data/data/vendor/atk4/dsql/src/Connection.php:365
/__w/data/data/src/Persistence/Sql.php:132
/__w/data/data/src/Model.php:1881
/__w/data/data/src/Model.php:1592
/__w/data/data/tests/SubTypesTest.php:34
/__w/data/data/tests/SubTypesTest.php:151
```